FiveCalls/FiveCalls.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
is this file change intentional?

Reply:
Yep, must have been a Xcode 15 change, I keep seeing it pop up in other branches